lib/generators/declare_schema/migration/migrator.rb

```diff
@@ -564,7 +564,7 @@ def change_foreign_key_constraints(model, old_table_name)
         end
 
         def remove_foreign_key(old_table_name, fk_name)
-          "remove_foreign_key('#{old_table_name}', name: '#{fk_name}')"
+          "remove_foreign_key(#{old_table_name.inspect}, name: #{fk_name.inspect})"
```

Comment:
This could be `fk_name.to_s.inspect` if we wanted to be consistent with `fk_name = nil` appearing as `name: ""`.

Reply:
I was looking into this behavior as it did seem odd. Thanks, this fix looks great.